.github/workflows/stale.yml

```diff
@@ -17,10 +17,14 @@ jobs:
         with:
           # Comma separated list of labels that can be assigned to issues to exclude them from being marked as stale
           exempt-issue-labels: contributions welcome, feature request, regression
+          # Override exempt-all-assignees but only to exempt the issues with an assignee to be marked as stale automatically
+          exempt-all-issue-assignees: true
```

Comment:
Looking at the existing stale issues with assignees, do they seem to still be active and worth keeping around?